transcendence-app/src/user/user.controller.ts

```diff
@@ -202,23 +209,6 @@ export class UserController {
     }
   }
 
-  @Get('block/:userId')
-  @HttpCode(HttpStatus.NO_CONTENT)
-  @ApiNoContentResponse({
-    description: 'Check if a user is blocked by the authenticated user',
-  })
-  @ApiBadRequestResponse({ description: 'Bad Request' })
-  @ApiNotFoundResponse({ description: 'Not Found' })
-  async getBlock(
-    @GetUser() user: User,
-    @Param('userId', ParseUUIDPipe) blockedUserId: string,
-  ): Promise<void> {
-    const block = await this.userService.getBlock(user.id, blockedUserId);
-    if (!block) {
-      throw new NotFoundException();
-    }
-  }
-
```

Comment:
I've deleted this method because of the approach taken. I didn't see a use case where we could want to get the relation of blocked uuids, that info is tied to users and would require some later logic...

Reply:
Our backend will need to be aware of the opposite situation, which is, when we actually implement the block, the authenticated user's session will need to know, not if they are blocking someone, but if someone blocked them. But obviously this shouldn't be transparent to users. And also this doesn't make your comment not true.
I don't know yet how will be implement this. But we will see.

Reply:
Agree, we will have to calculate that in backend